content/en/docs/reference/issues-security/issues.md

```diff
@@ -8,6 +8,7 @@ To report a security issue, please follow the [Kubernetes security disclosure pr
 
 Work on Kubernetes code and public issues are tracked using [GitHub Issues](https://github.com/kubernetes/kubernetes/issues/).
 
-* [CVE-related issues](https://github.com/kubernetes/kubernetes/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3Aarea%2Fsecurity+in%3Atitle+CVE)
+* [Official List of known CVEs](./../official-cve-feed) announced by [Security Response Committee](https://github.com/kubernetes/committee-security-response)
```

Comment:
```suggestion
* Official [list of known CVEs](/docs/reference/issues-security/official-cve-feed/)
  (security vulnerabilities) that have been announced by the
  [Security Response Committee](https://github.com/kubernetes/committee-security-response)
```
?

The key change here is to use a site-relative path rather than `./../official-cve-feed`

Reply:
I always miss this :( Thanks for catching it